src/marketplace/sellers/technical-review-guidelines.md

```diff
@@ -32,6 +32,48 @@ The Marketplace coding standard review uses a custom set of coding sniffs. If th
 
 _See also:_ [Coding Standards]({{ site.baseurl }}/guides/v2.3/coding-standards/bk-coding-standards.html)
 
+### Package Validation: verify that submitted code is a valid Magento extension
+
+This check verifies that submitted code:
+
+-  Is packaged as a valid Magento module, theme, language package, or meta-package
+-  Enforces best practices for Magento code distribution
+-  Helps to avoid common pitfalls
+
+Any code submitted for technical review at [Magento Developers Portal](https://developer.magento.com/) is examined to ensure that:
+
+1. Code submitted as a zip archive.
+1. Submitted package does not exceed 30 MB.
+1. Submitted package contains a `composer.json` file.
+1. The `composer.json` file specifies:
+   -  `name`
+   -  `type`
+   -  `version`
+1. The `composer.json` does not declare:
+   -  `extra.map`
+   -  `extra.magento-root-dir`
+1. The package has valid Composer package type:
+   -  for extensions, valid types are: `magento2-module`, `magento2-language`, `metapackage`
+   -  for themes, valid types is only `magento2-theme`
+   -  for shared libraries, valid types are: `magento2-module`, `magento2-theme`, `magento2-language`
+1. The package does not restict a set of compatible PHP versions declared by supported Magento versions.
```

Comment:
It's not critical, but it might be useful to attach a link to the corresponding page with a list of supported PHP versions.
https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements-tech.html#php

Reply:
In fact, we use information from [Magento composer file](https://github.com/magento/magento2/blob/2.3.4/composer.json#L14) which is simpler to parse than text from DevDocs. As so, for Magento 2.3 we now require compatibility with PHP 7.1.4+ even it is not recommended due to security concerns. 

We have an internal discussion on changing this rule to allow extensions to drop outdated PHP versions which cannot be removed from Magento because of backward compatibility restriction. Here we documented the current state.

As now these rules are documented we are open for discussion on how to improve them. Our goal is to increase the quality and interoperability of extensions, simplify submission process on the Developers portal and make it more transparent.

Reply:
@vkublytskyi, it's good to know, thank you for the details!